Package.swift

```diff
@@ -17,6 +17,7 @@ import PackageDescription
 
 let swiftSettings: [SwiftSetting] = [
     .enableUpcomingFeature("ExistentialAny"),
+    .enableExperimentalFeature("StrictConcurrency=complete"),
```

Comment:
I am extremely nervous about enabling this feature here: I'd prefer we did it in CI but left it off in the build.

Reply:
Removed.